lib/glimesh_web/controllers/user_settings_controller.ex

```diff
@@ -35,11 +37,68 @@ defmodule GlimeshWeb.UserSettingsController do
     end
   end
 
+  def create_channel(conn, _params) do
+    user = conn.assigns.current_user
+
+    case Streams.create_channel(user) do
+      {:ok, channel} ->
+        conn
+        |> put_session(:user_return_to, Routes.user_settings_path(conn, :stream))
+        |> UserAuth.log_in_user(user)
```

Comment:
Huh? should this be here?

Reply:
Currently use it to "reload" the page with the updated information. I'm probably missing a better function to do that.